src/pocketmine/command/defaults/XpCommand.php

```diff
@@ -58,38 +58,30 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
                         }
                 }
                 
-                if(strtolower(substr($args[0], -1)) == "l"){
-                        $lxp = $this->getInteger($sender, substr_replace($args[0], "", -1));
+                if(strcasecmp(substr($args[0], -1), "l") == 0){
```

Comment:
Why change this?

Reply:
I thought it would be a better comparison

Reply:
it's a completely unnecessary alteration.